lib/datahub-client/data_platform_catalogue/client/parsers.py

```diff
@@ -430,11 +428,11 @@ def parse_relations(
                     if relation.get("entity", {}).get("properties") is not None
                     else relation.get("entity").get("name", "")
                 )
-                description = (
-                    relation.get("entity").get("properties", {}).get("description", "")
-                    if relation.get("entity", {}).get("properties") is not None
-                    else ""
-                )
+                properties = relation.get("entity", {}).get("properties")
+                if properties is not None:
+                    description = properties.get("description") or ""
+                elif properties is None:
+                    description = ""
```

Comment:
We have tested for not None in the `if`
```suggestion
                else:
                    description = ""
```